### DISCUSSION TITLE
SUDO apt update fails accessing pi4j files

### DISCUSSION THREAD

**Discussion by lburaggi** [Pi4J/pi4j#88] [Q&A]
Running _sudo apt update_ I get the following error:
Failed to fetch https://repository.pi4j.com/dists/wheezy/InRelease  Could not resolve 'repository.pi4j.com'

and actually the url doesn't exist (anymore ?)

Has been moved recently ?

**Comment by savageautomate** [comment]
Yes it was moved with the release of v1.4 and preparation for v2.0.  However, we probably should look at redirecting the URL to avoid breaking.  

In the meantime, you can update your 
```
/etc/apt/sources.list.d/pi4j.list
```

to:
```
#########################################
# THE Pi4J PROJECT (V1) - APT REPOSITORY
#########################################
deb [arch=all] https://pi4j.com/download v1 stable
#deb [arch=all] https://pi4j.com/download v1 stable testing
```
(https://pi4j.com/v1/pi4j.list)

Or just uninstall and re-install using the `curl` methods listed here:
https://pi4j.com/1.4/install.html

Thanks, Robert
